public/scripts/open-project.js

```diff
@@ -45,9 +49,17 @@ require(["jquery"], function($) {
   Array.prototype.forEach.call(projects, function(project) {
     var projectSelector = "#" + project.getAttribute("id");
     var lastEdited = project.getAttribute("data-project-date_updated");
+    var projectID = project.getAttribute("data-project-id");
+
+    if (Lock.getLock(projectID)) {
+      $(projectSelector).addClass("disabled");
+      $(projectSelector + "> td > .project-delete").addClass("hide");
+      $(projectSelector + " .project-information").text("Project locked. Is it open in another tab?");
```

Comment:
I don't think you should expose the concept of the lock to the user (i.e., don't say "Project locked").  Instead, say something like, "The same project can't be opened more than once" or the like.